tests/testthat/test-profile_emissions.R

```diff
@@ -586,6 +511,7 @@ test_that("`profile_ranking_avg` is calculated correctly for benchmark `all`", {
 })
 
 test_that("yield NA in `*tilt_sector` and `*tilt_subsector` in *profile$ risk column", {
+  skip("FIXME: The test-data is incompatible with a new check")
```

Comment:
FIXME? I'm not sure what we're testing here, but the exact test-data we're using has NAs in all bencharks, and that's incompatible with this PR. 

TODO: Find out what we're testing and figure out a way to test it with a dataset compatible with this PR.

Reply:
@maurolepore Title of the test is not correct. My mistake! This test is testing that if `*profile$` column has NA then `tilt_sector` and `tilt_subsector` should always have NA. If you think this test is incompatible with this PR, then please remove it!